.github/workflows/deploy-pages.yaml

```
@@ -13,6 +13,11 @@ on:
         required: true
         type: string
         default: 'api-docs'
+      security_artifact:
+        description: 'Name of the security reports artifact'
+        required: true
+        type: string
+        default: 'security-reports'
```

actionlint (1.7.4)

input "security_artifact" of workflow_call event has the default value "security-reports", but it is also required. if an input is marked as required, its default value will never be used

(events)